src/main/java/org/jenkinsci/plugins/casc/plugins/PluginManagerConfigurator.java

```diff
@@ -102,8 +94,9 @@ public PluginManager configure(CNode config) throws ConfiguratorException {
                 plugins.add(new PluginToInstall(entry.getKey(), entry.getValue().asScalar().getValue()));
             }
         }
-
-        File shrinkwrap = new File("./plugins.txt");
+        
+        Optional<String> jenkinsRoot = Optional.ofNullable(System.getProperty("JENKINS_HOME", System.getenv("JENKINS_HOME")));
```

Comment:
You already have a Jenkins instance variable that is fully accessable in that section so consider using `jenkins.getRootDir()` 👍

Reply:
Good idead :) ~ Makes it simpler